1906-10-15.xml

```diff
@@ -0,0 +1,5812 @@
+                        <p><name>SCHLESWIG</name> leaves Alexandria 3 p.m. 14 &amp; 28 November; 12
+                            December.</p>
+                        <p><name>HOHENZOLLERN</name> leaves Alexandria 3 p.m. 24 October; 7 &amp; 31
+                            November; 5 December.</p>
+                        <p>The following steamers are intended to leave PORT-SAID:</p>
+                        <table rows="8" cols="6">
+                            <row>
+                                <cell cols="3">HOMEWARD : for Bremen Hamburg via Naples, Genoa,
+                                    (Gibraltar), Southampton, Antwerp.</cell>
+                            </row>
+                            <row>
+                                <cell><name>Scharnhorst</name></cell>
+                                <cell><measure quantity="8131" unit="ton">8131</measure> Tons</cell>
+                                <cell>about <date when="1906-10-21">19 Oct.</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Prinz Heinrich</name></cell>
+                                <cell><measure quantity="6263" unit="ton">6263</measure> Tons</cell>
+                                <cell>about <date when="1906-10-21">19 Oct.</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Gueiseuau</name></cell>
+                                <cell><measure quantity="6263" unit="ton">8081</measure> Tons</cell>
+                                <cell>about <date when="1906-10-21">3 Nov.</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Roon</name></cell>
+                                <cell><measure quantity="6263" unit="ton">5005</measure> Tons</cell>
+                                <cell>about <date when="1906-10-21">29 Nov.</date></cell>
+                            </row>
+                            <row>
+                                <cell cols="3">OUTWARD: for CHINA and JAPAN via SUEZ, ADEN, COLOMBO,
+                                    PENANG, SINGAPORE.</cell>
+                            </row>
+                            <row>
+                                <cell><name>Bulow</name></cell>
+                                <cell><measure unit="ton" quantity="8022">8000</measure> Tons</cell>
+                                <cell>about <date when="1906-10-01">15 Oct.</date></cell>
+                            </row>
+                            <row>
+                                <cell><name>Pr. Reg. Luitpold</name></cell>
+                                <cell><measure unit="ton" quantity="8000">6288</measure> Tons</cell>
+                                <cell>about <date when="1906-10-15">15 Oct.</date></cell>
+                            </row>
+                            <row>
+     
```

Comment:
contents of Personal and Social don't have to be items. They can just be paragraphs. This div should go all the way to 2560.